t/stream-node/healthcheck.t

```diff
@@ -0,0 +1,290 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+BEGIN {
+    if ($ENV{TEST_EVENTS_MODULE} ne "lua-resty-events") {
+        $SkipReason = "Only for lua-resty-events events module";
+    }
+}
```

Comment:
Is this necessary? If lua-resty-worker-event is used, events can't cross subsystems?

Reply:
For lua-resty-events, a second request need to be initiated to trigger the passive health check. Ref: https://github.com/apache/apisix/blob/1bac43a0517a675e4fe77145c2ad3d26f61033a0/t/node/healthcheck-passive-resty-events.t#L346.

For lua-resty-worker-events, when I add second request the test case is timing out, So the test case will be a litlte different based on which library is used. So I pinned the test to only lua-resty events.

Reply:
Do we need additional test cases to make sure it works? As I understand it, the current use cases with special handling can also override the lua-resty-worker-event library? 🤔

Reply:
lua-resty-worker-event case requires a little different handling in test. Okay I will separate these into two tests.